server/clearbit.js

```diff
@@ -30,31 +35,41 @@ export default class Clearbit {
     };
 
     if (api_key) {
-      this.client = new Client(api_key, this.metric, this.log);
+      this.client = new Client(api_key, this.metric, this.clearbitLogger);
     }
   }
 
-  log = (msg, data = "") => {
-    this.hull.logger.info(msg, data);
-  }
+  clearbitLogger = (msg, data = {}) => {
```

Comment:
Sure, but should all these logs stay at info level ? Because we have for example log as follows:
`outgoing.user.start` and that was at info level. Should I change level to `debug` or stick to `info` and add it to README ?